src/sbt-test/sbt-scalafix/relax-scalacOptions/test

```diff
@@ -1,22 +1,34 @@
-# we have 2 fatal warnings preventing compilation
+# check that regular compilation benefits from prior scalafix invocation
+# when the build does not contain any flag relaxed on scalafix invocation
+-> scalafixAll --check RemoveUnused
+-> checkLastCompilationCached
+> compile
+> checkLastCompilationCached
```

Comment:
this is the main driver behind the change: users with no fatal warnings calling scalafix before compile should not have to compile everything twice